walkthroughs/week-5-tdd/project/src/main/webapp/script.js

```diff
@@ -24,14 +24,16 @@ function sendMeetingRequest() {
   // split it into an array of names
   const attendees = attendeesNamesString.split(/\s*,\s*/);
 
-// comma-separated list of names
-  const optionalAttendeesNamesString = document.getElementById('optional-attendees').value;
+  // comma-separated list of names
+  const optionalAttendeesNamesString =
+      document.getElementById('optional-attendees').value;
```

Comment:
indentation should be 2 spaces

Reply:
Actually clang-format does work here, and the reason it changed this is because the line length was too long.

Reply:
So this is in fact correct.

Reply:
As below, this is a result of clang-format because the line length was too long.